content/en/docs/installation/helm.md

```diff
@@ -0,0 +1,174 @@
+---
+title: "Helm"
+linkTitle: "Helm"
+weight: 22
+type: "docs"
+---
+
+> Note: From cert-manager `v1.2.0` onward, the minimum supported version of
+> Kubernetes is `v1.16.0`. Users still running Kubernetes `v1.15` or below should
+> upgrade to a supported version before installing cert-manager.
+
+> **Warning**: You should not install multiple instances of cert-manager on a single
+> cluster. This will lead to undefined behavior and you may be banned from
+> providers such as Let's Encrypt.
+
+## Installing with Helm
+
+> **Note**: cert-manager should never be embedded as a sub-chart into other Helm charts.
+> cert-manager manages non-namespaced resources in your cluster and should only be installed once.
```

Comment:
suggestion: I don't think this needs to be formatted as a note, and we could add a little more intro text:

```suggestion
cert-manager provides Helm charts as a first-class method of installation on both Kubernetes and OpenShift.

Be sure never to embed cert-manager as a sub-chart of other Helm charts; cert-manager manages
non-namespaced resources in your cluster and care must be taken to ensure that it is installed exactly once.
```